<launch>

	<!-- AprilTag3 Detection -->
	<node	pkg="beginner_tello_application" 
		name="apriltag_detection" 
		type="apriltag_detection.py"> 
		<!-- output="screen"> -->
	</node>

</launch>
